Language/Functions/Analog IO/analogWrite.adoc

```diff
@@ -17,24 +17,28 @@ subCategories: [ "Analog I/O" ]
-| Nano 33 IoT *             | 2, 3, 5, 6, 9 - 12, A2, A3, A5 | 732 Hz
-| Nano 33 BLE/BLE Sense     | 1 - 13, A0 - A7                | 500 Hz
-| Due **                    | 2-13                           | 1000 Hz
-| 101                       | 3, 5, 6, 9                     | pins 3 and 9: 490 Hz, pins 5 and 6: 980 Hz
-|========================================================================================================
-{empty}* In addition to PWM capabilities on the pins noted above, the MKR, Nano 33 IoT, and Zero boards have true analog output when using `analogWrite()` on the `DAC0` (`A0`) pin. +
-{empty}** In addition to PWM capabilities on the pins noted above, the Due has true analog output when using `analogWrite()` on pins `DAC0` and `DAC1`.
+|=========================================================================================================================
+| Board                                      | PWM Pins *                     | PWM Frequency
+| UNO (R3 and earlier), Nano, Mini           | 3, 5, 6, 9, 10, 11             | 490 Hz (pins 5 and 6: 980 Hz)
+| UNO R4 (Minima, WiFi)                      | 3, 5, 6, 9, 10, 11             | 490 Hz
+| Mega                                       | 2 - 13, 44 - 46                | 490 Hz (pins 4 and 13: 980 Hz)
+| GIGA R1                                    | 2 - 13                         |
+| Leonardo, Micro, Yún                       | 3, 5, 6, 9, 10, 11, 13         | 490 Hz (pins 3 and 11: 980 Hz)
+| UNO WiFi Rev2, Nano Every                  | 3, 5, 6, 9, 10                 | 976 Hz
+| MKR boards **                              | 0 - 8, 10, A3, A4              | 732 Hz
+| MKR1000 WiFi **                            | 0 - 8, 10, 11, A3, A4          | 732 Hz
+| Zero **                                    | 3 - 13, A0, A1                 | 732 Hz
+| Nano 33 IoT **                             | 2, 3, 5, 6, 9 - 12, A2, A3, A5 | 732 Hz
+| Nano 33 BLE/BLE Sense                      | 1 - 13, A0 - A7                | 500 Hz
+| Due \***                                   | 2-13                           | 1000 Hz
+| 101                                        | 3, 5, 6, 9                     | pins 3 and 9: 490 Hz, pins 5 and 6: 980 Hz
+|=========================================================================================================================
+{empty}* These pins are officially supported PWM pins. While some boards have additional pins capable of PWM, using them is recommended only for advanced users that can account for timer availability and potential conflicts with other uses of those pins.  +
+{empty}** In addition to PWM capabilities on the pins noted above, the MKR, Nano 33 IoT, and Zero boards have true analog output when using `analogWrite()` on the `DAC0` (`A0`) pin. +
+{empty}pass:specialcharacters[***] In addition to PWM capabilities on the pins noted above, the Due has true analog output when using `analogWrite()` on pins `DAC0` and `DAC1`.
```

Comment:
```suggestion
|=========================================================================================================================
| Board                                      | PWM Pins +*+                   | PWM Frequency
| UNO (R3 and earlier), Nano, Mini           | 3, 5, 6, 9, 10, 11             | 490 Hz (pins 5 and 6: 980 Hz)
| UNO R4 (Minima, WiFi)                      | 3, 5, 6, 9, 10, 11             | 490 Hz
| Mega                                       | 2 - 13, 44 - 46                | 490 Hz (pins 4 and 13: 980 Hz)
| GIGA R1                                    | 2 - 13                         |
| Leonardo, Micro, Yún                       | 3, 5, 6, 9, 10, 11, 13         | 490 Hz (pins 3 and 11: 980 Hz)
| UNO WiFi Rev2, Nano Every                  | 3, 5, 6, 9, 10                 | 976 Hz
| MKR boards +**+                            | 0 - 8, 10, A3, A4              | 732 Hz
| MKR1000 WiFi +**+                          | 0 - 8, 10, 11, A3, A4          | 732 Hz
| Zero +**+                                  | 3 - 13, A0, A1                 | 732 Hz
| Nano 33 IoT +**+                           | 2, 3, 5, 6, 9 - 12, A2, A3, A5 | 732 Hz
| Nano 33 BLE/BLE Sense                      | 1 - 13, A0 - A7                | 500 Hz
| Due +***+                                  | 2-13                           | 1000 Hz
| 101                                        | 3, 5, 6, 9                     | pins 3 and 9: 490 Hz, pins 5 and 6: 980 Hz
|=========================================================================================================================
+*+ These pins are officially supported PWM pins. While some boards have additional pins capable of PWM, using them is recommended only for advanced users that can account for timer availability and potential conflicts with other uses of those pins.  +
+**+ In addition to PWM capabilities on the pins noted above, the MKR, Nano 33 IoT, and Zero boards have true analog output when using `analogWrite()` on the `DAC0` (`A0`) pin. +
+***+ In addition to PWM capabilities on the pins noted above, the Due has true analog output when using `analogWrite()` on pins `DAC0` and `DAC1`.
```

Previously three different approaches were using to prevent the `*` in this content from being treated as AsciiDoc markup:

- [backslash escaping](https://docs.asciidoctor.org/asciidoc/latest/subs/prevent/#escape-with-backslashes)
- [`empty` character replacement attribute]https://docs.asciidoctor.org/asciidoc/latest/attributes/character-replacement-ref/)
- [inline `pass` macro](https://docs.asciidoctor.org/asciidoc/latest/pass/pass-macro/#inline-pass)

I think this inconsistency in the markup makes it very difficult for anyone but an AsciiDoc expert to understand what is going on in this content. So I am proposing that a single approach be used everywhere.

I arbitrarily chose [the "single plus passthrough" syntax](https://docs.asciidoctor.org/asciidoc/latest/pass/pass-macro/#single-double-plus) simply because it is sufficient for this application and listed first in the documentation. But I don't mind if you prefer to use one of the other passthrough syntaxes instead.

I added the "passthroughs" to all the occurrences of `*` in the content even though it seems to not be necessary everywhere simply because I think it is more confusing for a contributor to try to understand exactly where the passthroughs are needed.

Reply:
Thank you @per1234. I'm new to AsciiDoc, you suggestion makes sense.